### ISSUE TITLE
Zest - It's possible to drag the script node

### ISSUE THREAD

**Issue by thc202** [zaproxy/zaproxy#2302]
Steps to reproduce the issue:
1. Run ZAP with Zest add-on installed;
2. Create a Zest Proxy script, with "Proxy default template.zst";
3. Select the script node and try dragging it;
4. Note that it shows a different icon indicating that it can not be dropped (which means it's being dragged);
5. Also, in the console it shows a lot of exceptions, for example:
```
java.lang.ClassCastException: org.zaproxy.zap.extension.zest.ZestScriptWrapper cannot be cast to org.zaproxy.zap.extension.zest.ZestElementWrapper
	at org.zaproxy.zap.extension.zest.ZestTreeTransferHandler.canImport(ZestTreeTransferHandler.java:66)
	at org.zaproxy.zap.extension.scripts.ScriptTreeTransferHandler.canImport(ScriptTreeTransferHandler.java:105)
	at javax.swing.TransferHandler$DropHandler.handleDrag(TransferHandler.java:1467)
	at javax.swing.TransferHandler$DropHandler.dragEnter(TransferHandler.java:1486)
	at java.awt.dnd.DropTarget.dragEnter(DropTarget.java:358)
```

Reproduced with:
 - ZAP 2.4.3;
 - "Zest - Graphical Security Scripting Language" version 19, older versions might also be affected.

**Comment by psiinon** [comment]
Hum, can fix the errors logged easily enough. Not so sure about making the script nodes undraggable, but will look into it...

**Comment by thc202** [comment]
This seems to only happen with Zest script nodes, because of https://github.com/zaproxy/zap-extensions/blob/04290932390a020c9a97a5fba96a05dd204d1ee5/src/org/zaproxy/zap/extension/zest/ExtensionZest.java#L199 ?
Is it really needed? Am I missing a use case for that?

**Comment by psiinon** [comment]
I cant think of one!
Added that fix to the PR :)

**Comment by thc202** [comment]
Fixed in zaproxy/zap-extensions#263.
